src/web/documentation/documentation.ts

```diff
@@ -0,0 +1,185 @@
+import * as vscode from 'vscode';
+import {YamlUtils} from '../libs/yamlUtils';
+import Markdown from '../libs/markdown';
+import ApiClient from '../apiClient';
+import taskHome from './task_home.md';
+import basic from './basic.md';
+
+let channel = vscode.window.createOutputChannel("Kestra - Docs");
```

Comment:
Another minor thing, I think this is potentially to be encapsulated within DocumentationPanel class to avoid use of global state.